### PR TITLE
[Fundamentals] Added page on site protection options

### PR DESCRIPTION
**Pull request by kodster28**
Pulling info from https://support.cloudflare.com/hc/en-us/articles/115002059131 to address PCX-3211

### CODE REVIEW

**Comment by kodster28** [comment]
From looking at the docs and stuff, it looks like the one-click `Protect my login` option isn't available in the newer Rate Limiting rules?

Or is that incorrect?

File: content/fundamentals/get-started/task-guides/secure-your-website.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- [Challenge Passage](https://support.cloudflare.com/hc/articles/200170136#2dwCrNWIMnNJDP6AVjEQ3e): Specify the length of time that a visitor can access your website after completing a security challenge.
+- [Browser Integrity Check](https://support.cloudflare.com/hc/articles/200170086): Evaluate incoming HTTP headers based on known threats — such as requests with a missing or non-standard user agent — and present a challenge page if needed.
+- [Privacy Pass Support](https://support.cloudflare.com/hc/articles/115001992652): Reduce the number of challenges presented to visitors using the **Privacy Pass** browser extension.
+
+### Protection with minimal setup
+
+Based on additional knowledge about your website traffic and requirements, you can also:
+
+- Enable [bot protection](/bots/get-started/)
+- Set up various **Security** rules:
```
Comment (kodster28) [inline content/fundamentals/get-started/task-guides/secure-your-website.md:34]: For now, this section isn't the _most_ helpful, but I think it's good enough? 

I've got an open question posed at https://chat.google.com/room/AAAAVBNahTM/Ei_Ulyj5gKo that'd help disambiguate between these different options. Once we get that answered, we can offload all the rules to a separate page and link to it.

File: content/fundamentals/get-started/task-guides/secure-your-website.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Secure your website
+
+Cloudflare offers several tools to protect your website against malicious traffic and bad actors.
+
+## Protection options
+
+### Default protection
+
+So long as your traffic is [proxied by Cloudflare](/fundamentals/get-started/concepts/how-cloudflare-works/), Cloudflare automatically protects your application from [DDoS attacks](https://support.cloudflare.com/hc/articles/200172676).
```
Comment (pedrosousa) [inline content/fundamentals/get-started/task-guides/secure-your-website.md]:
```suggestion
As long as your traffic is [proxied by Cloudflare](/fundamentals/get-started/concepts/how-cloudflare-works/), Cloudflare automatically protects your application from [DDoS attacks](https://support.cloudflare.com/hc/articles/200172676).
```
(Less informal, I think?)

File: content/fundamentals/get-started/task-guides/secure-your-website.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- [Security level](https://support.cloudflare.com/hc/articles/200170056): Use the IP reputation of a visitor to determine whether to present a [Managed Challenge](https://support.cloudflare.com/hc//articles/200170136#managed-challenge) page.
+- [Challenge Passage](https://support.cloudflare.com/hc/articles/200170136#2dwCrNWIMnNJDP6AVjEQ3e): Specify the length of time that a visitor can access your website after completing a security challenge.
+- [Browser Integrity Check](https://support.cloudflare.com/hc/articles/200170086): Evaluate incoming HTTP headers based on known threats — such as requests with a missing or non-standard user agent — and present a challenge page if needed.
+- [Privacy Pass Support](https://support.cloudflare.com/hc/articles/115001992652): Reduce the number of challenges presented to visitors using the **Privacy Pass** browser extension.
+
+### Protection with minimal setup
+
+Based on additional knowledge about your website traffic and requirements, you can also:
+
+- Enable [bot protection](/bots/get-started/)
```
Comment (pedrosousa) [inline content/fundamentals/get-started/task-guides/secure-your-website.md]:
```suggestion
- Enable [bot protection](/bots/get-started/).
```

File: content/fundamentals/get-started/task-guides/secure-your-website.md (hunk 1 above)
Comment (kodster28) [reply content/fundamentals/get-started/task-guides/secure-your-website.md:34]: And.... technically the answer from a lot of folks is we have other plans for certain rule types, so probably don't need to disambiguate between options for the moment.

Comment (pedrosousa) [reply content/fundamentals/get-started/task-guides/secure-your-website.md:34]: Yes, I think it's good as it is!

**Comment by pedrosousa** [comment]
> From looking at the docs and stuff, it looks like the one-click `Protect my login` option isn't available in the newer Rate Limiting rules?
> 
> Or is that incorrect?

You're right, it doesn't exist.

File: content/fundamentals/get-started/task-guides/secure-your-website.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+- [API Shield](/api-shield/): Protect your API from malicious traffic by enforcing schema validation, detecting abuse patterns, and more.
+- [Magic Firewall](/magic-firewall/): Use Cloudflare's firewall-as-a-service (FWaaS) to protect office networks and cloud infrastructure with advanced, scalable protection.
+- [Magic Transit](/magic-transit/): Delivers network functions at Cloudflare scale — DDoS protection, traffic acceleration, and much more from every Cloudflare data center — for on-premise, cloud-hosted, and hybrid networks.
+- [Magic WAN](/magic-wan/): Securely connect any traffic source - data centers, offices, devices, cloud properties - to Cloudflare’s network and configure routing policies to get the bits where they need to go, all within one SaaS solution.
+- [Page Shield](/page-shield/): Monitor third-party scripts on your application and receive notifications when they have been compromised or are exhibiting malicious behavior. 
+
+## Priority order
+
+Cloudflare's security tools evaluate HTTP requests in a specific order. For more details, refer to [Order and priority](/firewall/cf-firewall-rules/order-priority/).
```
Comment (pedrosousa) [inline content/fundamentals/get-started/task-guides/secure-your-website.md]: I probably wouldn't link just yet to this page. Currently, it only includes products that are _not_ based on the Ruleset Engine, leaving out many products. We have a separate list for the execution order of Ruleset-Engine-based products: https://developers.cloudflare.com/ruleset-engine/reference/phases-list/

We need to work on a global execution order page that includes everything, and that page will probably live outside the Firewall Rules documentation (maybe in Fundamentals, also?).

**Review by pedrosousa — APPROVED** [review]

File: content/fundamentals/get-started/task-guides/secure-your-website.md (hunk 4 above)
Comment (kodster28) [reply content/fundamentals/get-started/task-guides/secure-your-website.md]: Coolio, just removed from the page.

**Pull request merged**